src/cmd.ts

```diff
@@ -153,6 +153,27 @@ function renderPage(
   return ejs.render(layout.contents, context);
 }
 
+function renderPageContentsForRSS(
+  page: PageType,
+  appBaseURL: string,
+) {
+  const renderer =  new marked.Renderer();
+  renderer.link = (href, title, text) => {
+    if (href && href.startsWith("/")) {
```

Comment:
I don't quite understand this logic. Can `href` be falsey? If so, what happens when it's falsey since we return a link with that falsey value as the href?

Reply:
The `marked` types for these parameters are all nullable, so this just silenced that warning.

Reply:
Why is appBaseUrl only used in the case where the `if` condition is true?

Reply:
This replaces "relative" links used in Markdown with their absolute urls. But we don't want to do this if you are linking to something external like `https://github.com/whatrocks`

Reply:
Ahh, got it.

I wonder if it'd be more clear expressing that in the inverse:

```js
if (href.startsWith("http")) {
  // ...
}
```

... assuming the only protocols it could have are http(s)

Reply:
In any event, seems like a helper fn like `getLinkHref()` might be nice here.

Also, if we expect href to always be a string (despite marked types), you can consider telling the compiler with the `as` keyword.

```typescript
const linkHref = getLinkHref(href as string);
```

Reply:
Excellent feedback. Thank you.